src/modules/Dashboard/features/Applet/Popups/DuplicatePopups/DuplicatePopups.tsx

```diff
@@ -145,7 +152,7 @@ export const DuplicatePopups = ({ onCloseCallback }: { onCloseCallback?: () => v
         encryption,
         displayName: getValues('name'),
       },
-    });
+    }).then(handleDuplicateSuccess);
```

Comment:
Let's trigger `handleDuplicateSuccess` in one place with succeeded logic is triggered, right after the row 91.